templates/footer.hbs

```diff
@@ -1,15 +1,23 @@
 <footer class="footer">
-  <h3 class="footer__title">{{navLinksTitle}}</h3>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4 offset-md-2">
+        <h3 class="footer__title">{{navLinksTitle}}</h3>
+        <ul class="footer__links">
+          {{#each navLinks}}
+            <li class="footer__link p">{{{module this}}}</li>
+          {{/each}}
+        </ul>
+      </div>
 
-  <ul class="footer__links">
-    {{#each navLinks}}
-      <li class="footer__link p">{{{module this}}}</li>
-    {{/each}}
-  </ul>
+      <div class="col-md-4 offset-1">
```

Comment:
change `offset-1` to `offset-md-1` to avoid having the offset on the mobile version. We need a margin-top to separate this div from the links on top.